Login/routes/patch1.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+
+const router = express.Router();
+
+HEROKU_CRED = process.env.HEROKU_CRED
```

Comment:
What does this contain, and if you are using it as a secret then why are you sending it out to the user in the `/auth2check` route.

Reply:
ok, will fix this